.github/CODEOWNERS

```
@@ -10,3 +10,6 @@
 
 # Julian contributed the OpenTelemetry implementation.
 /tracing-opentelemetry/    @jtescher @tokio-rs/tracing
+
+# Zeki contributed the TracingAppender implementation
+/tracing-appender/              @zekisherif @hawkw  @tokio-rs/tracing
```

I don't need to be assigned to `tracing-appender` PRs, since the `@tokio-rs/tracing` group will tag me anyway:
```suggestion
/tracing-appender/              @zekisherif  @tokio-rs/tracing
```